src/gtc/cuir/kernel_fusion.py

```diff
@@ -50,7 +50,8 @@ def is_parallel(kernel: cuir.Kernel) -> bool:
                 kernel.iter_tree()
                 .if_isinstance(cuir.FieldAccess)
                 .filter(
-                    lambda x: x.offset.i != 0 or x.offset.j != 0 or (x.offset.k != 0 and parallel)
+                    lambda x: any(off != 0 for off in x.offset.to_dict().values())
```

Comment:
Why this change?

Reply:
It didn't work with VariableKOffset on a test that I enabled.